lib/internal/streams/pipeline.js

```diff
@@ -0,0 +1,95 @@
+// Ported from https://github.com/mafintosh/pump with
+// permission from the author, Mathias Buus (@mafintosh).
```

Comment:
if the original code has a copyright/license, then it should likely either be included here or embedded in the LICENSE file.

Reply:
It does, but I did mention here I'm fine waiving copyright, https://github.com/mafintosh/pump/issues/17#issuecomment-299838662

Reply:
(I'm fine with whatever is easiest btw)